lib/rust/ensogl/core/src/display/navigation/navigator/events.rs

```diff
@@ -23,9 +23,15 @@ pub struct ZoomEvent {
 }
 
 impl ZoomEvent {
-    fn new(focus: Vector2<f32>, amount: f32, zoom_speed: f32) -> Self {
+    /// Constructor. Returns `None` if `amount` or `zoom_speed` are equal to zero.
```

Comment:
I'd be more specific here: Not the `zoom_speed` is _almost_ equal (in the range of `f32::Epsilon`).

Reply:
Fixed